src/sidebar/util/sentry.js

```diff
@@ -20,7 +20,8 @@ function currentScriptOrigin() {
   try {
     // nb. IE 11 does not support `document.currentScript` and this property
     // is only available while a `<script>` tag is initially being executed.
-    const scriptUrl = new URL(document.currentScript.src);
+    const script = /** @type {HTMLScriptElement} */ (document.currentScript);
+    const scriptUrl = new URL(script.src);
```

Comment:
Yes, exactly.

The type cast above is needed because it turns out that `document.currentScript` can be an HTML `<script>` tag or an SVG `<script>` tag and TS doesn't have the context to know what kind of document we are in.

Reply:
I wonder if that should be added as a comment, then, so we don't lose it?